doc/classes/TreeItem.xml

```diff
@@ -553,14 +560,24 @@
 				Sets the given column's custom color.
 			</description>
 		</method>
-		<method name="set_custom_draw">
+		<method name="set_custom_draw" is_deprecated="true">
 			<return type="void" />
 			<param index="0" name="column" type="int" />
 			<param index="1" name="object" type="Object" />
 			<param index="2" name="callback" type="StringName" />
 			<description>
 				Sets the given column's custom draw callback to [param callback] method on [param object].
 				The [param callback] should accept two arguments: the [TreeItem] that is drawn and its position and size as a [Rect2].
+				[i]Deprecated.[/i] Use [method TreeItem.set_custom_draw_callback] instead.
+			</description>
+		</method>
+		<method name="set_custom_draw_callback">
+			<return type="void" />
+			<param index="0" name="column" type="int" />
+			<param index="1" name="callback" type="Callable" />
+			<description>
+				Sets the given column's custom draw callback. Call with [code skip-lint]Callable()[/code] to clear the custom callback.
```

Comment:
Unsure about the exact wording here but think relevant to point out you can clear it and how to do so

Reply:
Maybe
```suggestion
				Sets the given column's custom draw callback. Use empty [Callable] ([code skip-lint]Callable()[/code]) to clear the custom callback.
```

Reply:
Will include in my next push